gtsam/hybrid/GaussianMixtureConditional.h

```diff
@@ -0,0 +1,92 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file   GaussianMixtureConditional.h
```

Comment:
In one of my reviews I asked it to be renamed to `GaussianMixtureConditional` so that it is obvious it is a conditional and contrasts with `GaussianMixtureFactor`. I can undo that if you'd like.

Reply:
I would like you to undo it, yes.